tsparser/src/parser/resources/apis/api.rs

```diff
@@ -184,7 +189,11 @@ pub const ENDPOINT_PARSER: ResourceParser = ResourceParser {
             let methods = r.config.method.unwrap_or(Methods::Some(vec![Method::Post]));
 
             let raw = matches!(r.kind, EndpointKind::Raw);
```

Comment:
Could arguably remove this variable now and use `true/false` since you moved the call to `describe_endpoint` into the `match`?

Reply:
we actually need the raw value further down when we create the `Endpoint`. I switched to `true/from` for the describe calls, as that is maybe a bit cleaner